lib/dao.ts

```diff
@@ -172,7 +172,7 @@ export class DAO {
    * @param address - optional
    */
   public async getContractWrapper(contract: string, address?: Address): Promise<ContractWrapperBase | undefined> {
-    return Contracts.getContractWrapper(contract, address);
+    return WrapperService.getContractWrapper(contract, address);
```

Comment:
also why call these things Service? seems a bit unnecessary of a word to me

Reply:
I am following a common pattern to define and name these classes according to their functionality as a "Service".